### ISSUE TITLE
Unable to update beta in playstore

### ISSUE THREAD

**Issue by vannaka** [SUPERCILEX/Robot-Scouter#128]
I've been unable to update the beta app since version 1.1.2.5415.

Here is a screen shot of the error message. 
<a href='http://i.imgur.com/1kWdZfz' title=''><img src='http://i.imgur.com/1kWdZfz.jpg' alt='' title='Hosted by imgur.com' /></a>

**Comment by blake1029384756** [comment]
I am having this same problem. It started this week sometime, I am not sure after which build. I have tried the solutions on Google and they don't work, wiping the playstore data just results in it crashing.  

I opted out of the beta program and am able to install the app again. All my other apps update just fine aswell

**Comment by SUPERCILEX** [comment]
@vannaka @blake1029384756 Lol, competition season is over, you guys shouldn't be worrying about this stuff! 😄 Thanks for report anyway. @dracco1993 also noticed so I'll share the conversation I had with him:
![image](https://cloud.githubusercontent.com/assets/9490724/26532175/c9652670-43ae-11e7-950b-8a72a501890a.png)

**TL;DR:** Can't downgrade because I already converted my project to Java 8 and can't upgrade, because there's a bug that prevents RS from compiling. So basically, I'm surrounded on all sides and I just have to wait it out. Here's the bug report I filed: https://issuetracker.google.com/issues/62148494.

**Comment by SUPERCILEX** [comment]
@vannaka @blake1029384756 @dracco1993 Hey guys, I finally got around to getting my release builds to work. Publishing a new release on every CI build is broken, but I've added the latest APK to the Play Store.

There's also another bug in the Gradle plugin that leads to missing class references. Specifically, the Android animator class is missing so if you're opening Robot Scouter for the first time or running some animation in the app, it'll crash with a class not found exception. 😕 Classic Google. 😆 

If the Play Store hasn't annoyed you enough to uninstall Robot Scouter since I broke it 😄, I've done some more UI revamps, fixed bugs, and added a night mode that will eventually be enabled/disabled from a settings screen. For now you can enable it by changing your phone's time to 11pm or something.

Thanks for your support! 😄

**Comment by SUPERCILEX** [comment]
Aw crap, I just tested the release build and it looks like proguard deleted my main fragment so the app will crash on startup everytime. Sigh. 😢 More waiting on Google to fix stuff.
